runtime/stateManager.py

```diff
@@ -83,6 +93,9 @@ def make_hibike_response_map(self):
         return {k.value: v for k, v in hibike_response_mapping.items()}
 
     def init_robot_state(self):
+        """
+        Initalizes StateManager in ROBOT state.
```

Comment:
Perhaps this should be instead, "Initialize robot state". Also, robot is not capitalized.